ios/engine/KMEI/KeymanEngineTests/resources/Queries.bundle/package-version-km-updated.json

```
@@ -0,0 +1,8 @@
+{
+    "keyboards": {
+        "khmer_angkor": {
+            "version": "1.1.0",
+            "kmp": "https://downloads.keyman.com/keyboards/khmer_angkor/1.0.6/khmer_angkor.kmp"
```

This does not match what I'd expect given the version string:
```suggestion
            "kmp": "https://downloads.keyman.com/keyboards/khmer_angkor/1.1.0/khmer_angkor.kmp"
```